release/models/gnpsi/openconfig-gnpsi.yang

```diff
@@ -0,0 +1,96 @@
+module openconfig-gnpsi {
+  yang-version "1";
+  namespace "http://openconfig.net/yang/gnpsi";
+  prefix "oc-gnpsi";
+
+  import openconfig-extensions {
+    prefix oc-ext;
+  }
+
+  import openconfig-system {
+    prefix oc-sys;
+  }
+
+  import openconfig-system-grpc {
+    prefix oc-sys-grpc;
+  }
+
+  import openconfig-gnpsi-types {
+    prefix oc-gnpsit;
+  }
+
+  import openconfig-inet-types {
+    prefix oc-inet;
+  }
+
+  import openconfig-yang-types {
+    prefix oc-yang;
+  }
+
+  organization
+    "OpenConfig Working group";
+  contact
+    "www.openconfig.net";
+
+  description
+    "This module defines configuration and operational state data
+    related to gRPC Network Packet Sampling Interface (gNPSI).
+
+    GNPSI Proto Reference: https://github.com/openconfig/gnpsi";
+
+  oc-ext:openconfig-version "0.1.0";
+  oc-ext:catalog-organization "openconfig";
+  oc-ext:origin "openconfig";
+
+  revision 2024-05-16 {
+    description
+      "Initial revision.";
+    reference "0.1.0";
+  }
+
+  grouping gnpsi-counters {
+    description
+      "Top-level container for gNPSI counters.";
+    container counters {
+      description
+        "Operational data for gNPSI counters.";
+      uses gnpsi-counters-top;
+    }
+  }
+
+  grouping gnpsi-counters-top {
+    description
+      "Top-level container of operational data for gNPSI counters.";
+
+    leaf bytes-sent {
+      type oc-yang:counter64;
+      description
+        "The total number of bytes sampled and sent to the
+        client.";
+    }
+
+    leaf packets-sent {
+      type oc-yang:counter64;
+      description
+        "The total number of packets sampled and sent to the
+        client.";
+    }
+
+    leaf packets-error {
+      type oc-yang:counter64;
+      description
+        "The total number of error packets failed to be sampled
+        and sent to the client.";
+    }
+  }
```

Comment:
packet/byte counters should also be moved to the gRPC state (and the error counter should probably be removed) since they're not specific to gNPSI streams anymore

Reply:
What about move packet/byte counters to gRPC state and leave error counter for gNPSI. gNPSI needs error counter

Reply:
agreed, the `packets-error` leaf is gnpsi specific

Reply:
Could you please elaborate on how this error counter is supposed to work?
What exactly is counted as an error?

Reply:
Any error when writing into gRPC stream would be considered as a packets-error.

Ref: https://github.com/search?q=repo%3Aopenconfig%2Fgnpsi%20IncrementWriteErrorCount()&type=code

Reply:
I have updated the PR as you suggested :)